index.dd

```diff
@@ -366,6 +367,23 @@ consistency. $(LINK2 spec/memory-safe-d.html, Read more).)
 
 )))
 
+$(DIVC section, $(DIV, $(SECTION3 Industry-proven,
+$(DIVC frontpage-orgs,
+    $(FRONTPAGE_ORG Facebook, https://www.facebook.com, facebook.svg,
+    )
+    $(FRONTPAGE_ORG eBay, https://ebay.com, ebay.svg,
+    )
+    $(FRONTPAGE_ORG Remedy Games, http://remedygames.com, remedy.svg,
+    )
+    $(FRONTPAGE_ORG Sociomantic, https://www.sociomantic.com, sociomantic.png,
+    )
+    <div id="frontpage-orgs-more">
+        $(LINK2 $(ROOT_DIR)/orgs-using-d.html, Show more organizations using D)
```

Comment:
`$(ROOT_DIR)orgs-using-d.html` (or `$(ROOT)/orgs-using-d.html`), not `$(ROOT_DIR)/orgs-using-d.html`

Also, no "Show" I think, just "More organizations using D". Technically most links are used to show something else.